tests/cypress/support/ExpressionEngineUpdate.pub

```diff
@@ -1,9 +0,0 @@
- - -----BEGIN PUBLIC KEY-----
```

Comment:
Did we just no longer need to include this public key?

Reply:
Correct - not needed (also not real public key)